apps/exchange/lib/exchange/matching_engine.ex

```diff
@@ -344,12 +344,16 @@ defmodule Exchange.MatchingEngine do
     if Enum.count(trades) > 0 do
       trades
       |> Enum.each(fn t ->
-        EventBus.cast_event(:trade_executed, %EventBus.TradeExecuted{trade: t})
+        message_bus_adapter().cast_event(:trade_executed, t)
       end)
 
       OrderBook.flush_trades!(order_book)
     else
       order_book
     end
   end
+
+  defp message_bus_adapter do
```

Comment:
we can rename this method as `message_bus()` and leave the notion that this is configurable as an adapter to the config file like:

```elixir
config :exchange, 
  message_bus_adapter: EventBus


and 
defp message_bus_adapter do
 Application.get_env(:exchange, :message_bus_adapter)
end
```

This way we'll  have `message_bus().cast_event(:trade_executed, t)` which is more clear and direct.